Documentation/docs/using-fluent.md

```diff
@@ -0,0 +1,220 @@
+    <Application.Resources>
+        <ResourceDictionary Source="pack://application:,,,/PresentationFramework.Fluent;component/Themes/Fluent.xaml" />
+    </Application.Resources>
+</Application>
+```
+
+Similary Fluent theme can be enabled for a particular WPF window by including the ResourceDictionary in Window's XAML file as follows:
+```xml
+<Window
+    x:Class="YourSampleApplication.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:local="clr-namespace:YourSampleApplication"
+    mc:Ignorable="d"
+    Title="MainWindow" Height="450" Width="800">
+    <Window.Resources>
+        <ResourceDictionary Source="pack://application:,,,/PresentationFramework.Fluent;component/Themes/Fluent.xaml" />
+    </Window.Resources>
+    <Grid>
+
+    </Grid>
+</Window>
+```
+
+By default, when `Fluent.xaml` is included it reacts to the system theme changes. If you want your application to be in Light or Dark mode only include the following dictionaries respectively:
+```xml
+    <!-- For Fluent Light mode -->
+    <ResourceDictionary Source="pack://application:,,,/PresentationFramework.Fluent;component/Themes/Fluent.Light.xaml" />
+
+    <!-- For Fluent Dark mode -->
+    <ResourceDictionary Source="pack://application:,,,/PresentationFramework.Fluent;component/Themes/Fluent.Dark.xaml" />
+```
+
+### Setting Fluent theme using the experimental ThemeMode APIs
+
+In .NET 9, we also provided experimental APIs : `Application.ThemeMode` and `Window.ThemeMode` which can be set from XAML or code-behind to enable the Fluent theme in WPF app.
+
+For setting ThemeMode on Application, you can do the following in App.xaml:
+```xml
+<Application 
+    x:Class="YourSampleApplication.App"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="clr-namespace:YourSampleApplication"
+    ThemeMode="Dark">
+    <Application.Resources>
+    
+    </Application.Resources>
+</Application>
+```
+
+Similary, we can set the ThemeMode on any Window as follows:
+```xml
+<Window
+    x:Class="YourSampleApplication.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:local="clr-namespace:YourSampleApplication"
+    mc:Ignorable="d"
+    Title="MainWindow" Height="450" Width="800" ThemeMode="Dark">
+    
+</Window>
+```
+
+Since, the API is experimental, for using `ThemeMode` property in code-behind, we need to suppress **WPF0001** warning first. This can be done in the application's project file as follows:
```

Comment:
You do not need to suppress the warning, what is the point of the warning if everyone suppresses it? I suggest the whole section is dropped, you can use the code-behind properties with a wawraning.